### PR TITLE
Add scrumble's label tracking tool

### PR DESCRIPTION
**Pull request by AmauryLiet**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-scrumble-tools/react-native/debugging/debug-javascript-ios-device.mo.md">debug-javascript-ios-device.mo.md</a>**: MO doesn't have an *Owner*, could it be you?
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-scrumble-tools/contributing/mo.s.md">mo.s.md</a>**: Title doesn't contain `[Standard]` tag at the begining
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-scrumble-tools/react-native/debugging/debug-javascript-ios-device.mo.md">debug-javascript-ios-device.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-scrumble-tools/project-tools/label-tracking.mo.md">label-tracking.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by tychota** [comment]
`git commit --amend` et `push -f` pour remove le label du client

**Comment by AmauryLiet** [comment]
TODO:
Change image to a simpler project (without the client name!)

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+.idea/
```
Comment (Minishlink) [inline .gitignore:2]: Should be in your global gitignore, not here ;)

File: contributing/mo.s.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# [Standard] Method of Operation
+# [MO] Method of Operation
```
Comment (Minishlink) [inline contributing/mo.s.md:1]: It's the Standard for writing a method of operation :) (so there shouldn't be any change here)

File: README.MD [hunk 3]
```diff
@@ -57,6 +57,9 @@ we help him solve his problem and then together write either a [Standard] or a [
 
 - [[MO] Setup VSCode](/editors/vscode/setup-vscode.mo.md)
 
+## Project tools
+- [[MO] Label tracking with scrumble](/project-tools/label-tracking.mo.md)
```
Comment (Minishlink) [inline README.MD:61]: Scrumble?

File: contributing/mo.s.md (hunk 2 above)
Comment (tychota) [reply contributing/mo.s.md:1]: +1

File: .gitignore (hunk 1 above)
Comment (tychota) [reply .gitignore:2]: I won't be so strong with it. 

It has no impact to add this to gitignore, plus in some project you want to version certain (but not all file of `./idea`).

Having those file to gitignore global is nice:
```
# User-specific stuff:
.idea/**/workspace.xml
.idea/**/tasks.xml
.idea/dictionaries

# Sensitive or high-churn files:
.idea/**/dataSources/
.idea/**/dataSources.ids
.idea/**/dataSources.xml
.idea/**/dataSources.local.xml
.idea/**/sqlDataSources.xml
.idea/**/dynamic.xml
.idea/**/uiDesigner.xml
```

Having the whole `.idea` should be project specific.

What do you think @Minishlink ?

Comment (Minishlink) [reply .gitignore:2]: +1

**Comment by yleflour** [comment]
Il manque toujours des éléments dans ton MO. Owner + le tag [Standard]

**Comment by tychota** [comment]
Closing as too old PR. Feel free to reopen later.

**Pull request closed without merging**